scripts/dig.js

```diff
@@ -0,0 +1,27 @@
+var b = pizza.open('https://toolbox.googleapps.com/apps/dig/');
+
+pizza.sleep(1000);
```

Comment:
Rather than a fixed sleep, you can just wait on thing you are going to click on:

b.waitForVisible('#domain');